src/test/java/org/takes/tk/TkSlf4jTest.java

```diff
@@ -55,4 +61,36 @@ public void logsException() throws IOException {
         new TkSlf4j(new TkFailure(new IOException(""))).act(new RqFake());
     }
 
+    /**
+     * {@link TkSlf4j} can output an empty body for {@link TkEmpty}.
+     * @throws IOException if some I/O problem occurred.
+     */
+    @Test
+    public void testEmptyResponseBody() throws IOException {
```

Comment:
@rui-castro The name of your unit test is not correct, it should start with a conjugated verb